config/software/berkshelf.rb

```diff
@@ -43,4 +43,20 @@
 
   gem "install pkg/berkshelf-*.gem" \
       " --no-ri --no-rdoc", env: env
+
+  if windows?
+    block "Clean up large object files" do
+      lib_path = File.join(
+        "#{install_dir.gsub(/\\/, '/')}",
+        "embedded/lib/ruby/gems/**/gems",
+        "dep-selector-libgecode-*",
+        "lib/dep-selector-libgecode/vendored-gecode/lib",
+      )
+
+      Dir.glob("#{lib_path}/*.a").each do |f|
+        puts "Deleting #{f}"
+        File.delete(f)
+      end
```

Comment:
Somehow I find this all a bit distracting to read.

```ruby
# find the embedded rubygems dir and clean it up for globbing
gem_dir = "#{install_dir}/embedded/lib/ruby/gems/*/gems".gsub(/\\/, '/')

# find all the static *.a files in the dep-selector-libgecode gem we don't use and delete them
Dir.glob("#{gem_dir}/dep-selector-libgecode*/**/*.a").each do |f|
  puts "Deleting #{f}"
  File.delete(f)
end
```

That breaks it up into two pieces and adds some comments explaining what you were doing because it wasn't initially obvious to me...

I also suspect that this code can be reused on non-windows (although you might have to !aix? it since aix is evil and puts dynamic object in .a files because its just a hateful, hateful operating system...)